docs/tutorials/proposals/referral-program-proposal.md

```diff
@@ -0,0 +1,226 @@
+---
+sidebar_position: 10
+title: Enable or replace referral program
+hide_title: false
+vega_network: TESTNET
+keywords:
+- proposal
+- governance
+---
+
+import NetworkParameter from '@site/src/components/NetworkParameter';
+import JSONInstructions from './_json-instructions.md';
+import TerminalInstructions from './_terminal-instructions.md';
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The on-chain referral program allows users to refer new traders. Referrers can get a cut of their referees' trading fees, and referees get a discount on their fees. In addition, having VEGA tokens associated can multiply participants' proceeds.
```

Comment:
> In addition, having VEGA tokens associated can multiply participants' proceeds.

Small nitpick, to me, this line implies staking tokens multiplies a referees discounts which isn't true, it only multiples referrers rewards.